### PR TITLE
Initial work to enable modules for tenants

### PR DESCRIPTION
**Pull request by LiamThursfield**
- This enables the field / UI for setting enabled modules
- Will require further work to make use of those modules to enable/disable features on the tenant side

image

image

### CODE REVIEW

File: app/Interfaces/TenantInterface.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Interfaces;
+
+class TenantInterface
+{
+    public const MODULE_CMS = 'CMS';
+    public const MODULE_CRM = 'CRM';
+    public const MODULE_EDU = 'EDU';
```
Comment (dorusomcutean) [inline app/Interfaces/TenantInterface.php:9]: Minor: Thinking we add REPORTS now as well? Is sth that will need adding soon anyway? Although it will be scope creep so nah

**Review by dorusomcutean — APPROVED** [review]

File: app/Interfaces/TenantInterface.php (hunk 1 above)
Comment (LiamThursfield) [reply app/Interfaces/TenantInterface.php:9]: Yah I think we could add the module here at the same time as reporting tbf!
